src/aleph/vm/storage.py

```diff
@@ -132,7 +132,7 @@ async def get_latest_amend(item_hash: str) -> str:
     if settings.FAKE_DATA_PROGRAM:
         return item_hash
     else:
-        url = f"{settings.CONNECTOR_URL}/compute/latest_amend/{item_hash}"
+        url = f"{settings.CONNECTOR_URL}compute/latest_amend/{item_hash}"
```

Comment:
would be nice to have something in the settings that add that / if it doesn't exist

Reply:
Yeah, I could add a validator for this method with something like this

```py
@validator("CONNECTOR_URL", pre=True)
    def ensure_trailing_slash(cls, value: str) -> str:
        if not value.endswith("/"):
            return f"{value}/"
        return value
```

This would add automatically a slash if missing after calling `settings.CONNECTOR_URL`

Reply:
Yep can do a function to do that but I also checked the documentation of Pydantic 2, the HttpUrl ensures that the argument given is well formated.
That way it will automatically add an extra slash if needed.

So is this really necessary to create a function knowing that pydantic will add automatically a slash?

Reply:
Didn't know pydantic did it but if that's the case no it is not needed